src/domain/registration/types/RegistrationTypes.tsx

```diff
@@ -1,7 +1,18 @@
 export interface RegistrationFormValues {
-  childBirthday: string;
-  childHomeCity: string;
+  child: {
```

Comment:
I think this file should be `.ts`-extension as well

Reply:
Wont hurt if with/without `x`
Sometime util return html -> x is wider choice

Reply:
but yea the typing should be ts